src/librustc/mir/mod.rs

```diff
@@ -1274,7 +1274,8 @@ pub struct VisibilityScopeData {
 /// being nested in one another.
 #[derive(Clone, PartialEq, RustcEncodable, RustcDecodable)]
 pub enum Operand<'tcx> {
-    Consume(Lvalue<'tcx>),
+    Copy(Lvalue<'tcx>),
```

Comment:
Can we leave some comments explaining what these mean, in terms of the expectations they provide? I think something like this:

- Copy: The value must be available for use afterwards. This implies that the type of the lvalue must be `Copy`; this is true by construction during build, but checked by the MIR type checker.
- Move: The value will not be used again. Safe for values of all types (modulo future developments towards `?Move`). Enforced by borrow checker. Sometimes Copy is converted to Move to enable "last-use" optimizations.